lib/main.coffee

```diff
@@ -29,7 +39,7 @@ module.exports =
 
     # Check version of coala
     version = helpers.exec(@executable, ['--version']).then (result) ->
-      if result <= COALA_MIN_VERSION
+      if cmpVersions(result, COALA_MIN_VERSION) < 0
```

Comment:
I did a little digging around, https://www.python.org/dev/peps/pep-0440/ the current recommended method of pre/post release tagging can be handled by `pkg_resources.parse_version()`.